packages/sync/README.md

```diff
@@ -17,3 +17,140 @@ To enable debug please execute in console
+  conflictStrategy: strategies.diffMergeClientWins,
+  customLinkBuilder: customLinkBuilder,
+  networkStatus: customNetworkStatus,
+  mutationsQueueName: "offline-mutations-queue",
+  squashing: true
+}
+```
+
+Config | Description | Default
+------ | ------ | ------
+httpUrl | The URL of your http server | N/A
+wsUrl | The URL of your websocket | N/A
+dataIdFromObject | The name of the field to be used as ID | "id"
+storage | The storage you want your client to use | window.localStorage
+conflictStrategy | The conflict resolution strategy your client should use | N/A
+customLinkBuilder | Enables providing custom Apollo Link for processing requests | See `LinksBuilder`
+networkStatus | Implementation of `NetworkStatus` Interface | See `WebNetworkStatus` and `CordovaNetworkStatus`
+mutationsQueueName | The name to store requests under in your offline queue | "offline-mutation-store"
+squashing | Whether or not you wish to squash mutations in your queue | true
+
+
+# Creating a Client
+```javascript
+let client = createClient(config);
+```
+
+
+## Conflict strategies
+You can provide your custom conflict resolution strategies to the client in the config by using the provided `ConflictResolutionStrategy` type from the SDK. An example custom strategy is shown below.
+
+```javascript
+let clientWins = (serverData, clientData) => {
+  return Object.assign(server, client);
+};
+```
+
+To use this strategy pass this function as conflictStrategy in your config object like so:
+
+```javascript
+let config = {
+...
+  conflictStrategy: clientWins
+...
+}
+```
+
+## Implementing Custom Apollo Links
+To use your own custom apollo links to create your own network flow, simply follow the documentation for creating links here: https://www.apollographql.com/docs/link/index.html. You can pass this building mechanism to your client in the config, under the `customLinkBuilder` parameter.
+
+```javascript
+export const linkBuilder: LinkChainBuilder = (): ApolloLink => {
+    const httpLink = new HttpLink({ uri: "someUri" });
+    const customLink = new YourCustomLink();
+
+    let links: ApolloLink[] = [customLink, httpLink];
+
+    let compositeLink = ApolloLink.from(links);
+    return compositeLink;
+  };
+```
+
+## Implementing Custom Network Status checks
+To use your own custom netork checks, implement the NetworkStatus interface which provides two functions;
+
+```javascript
+  onStatusChangeListener(callback: NetworkStatusChangeCallback): void;
+
+  isOffline(): boolean;
+```
+
+## Offline Mutations
+
+AeroGear Sync SDK provides offline mutations out of the box. By default it uses localStorage to do this and mutations are stored under the cache key of 'offline-mutations-queue'.
+
+### Online Only Queries
+To ensure certain queries are not queued and are always delivered to the network layer, you must make use of Graphql directives. To do so define a directive on your schema type definitions like so:
+
+```
+directive @onlineOnly on FIELD
```

Comment:
This is no longer problem . This should be defined on server - server should use graphql lower than 14. that allows that. 
I guess we could create ticket on the server and adjust docs accordingly.

WDYT?

Reply:
So lower than version 14 and you don't need to explicitly call it out? Should we not be using the most recent version? I have no preference either way.